### ISSUE TITLE
Links in system chat messages should have more contrast

### ISSUE THREAD

**Issue by gabek** [owncast/owncast#2634]
### Share your bug report, feature request, or comment.

It is now purple on purple.

![image](https://user-images.githubusercontent.com/414923/215028022-65457e49-8346-4fe6-98d3-23da5e06dd8c.png)
